### PR TITLE
[11.0] web_export_view: add security group

### PR DESCRIPTION
**Pull request by chienandalu**
- Add a security group to disallow users to export data from the view

cc @Tecnativa

### CODE REVIEW

**Comment by tarteo** [comment]
Is this https://github.com/OCA/web/tree/11.0/web_disable_export_group not the same?

**Comment by pedrobaeza** [comment]
No, that one is a complement. This one is restricting the new button at list level that this module adds that allows to export directly "what you see".

File: web_export_view/static/src/js/web_export_view.js [hunk 1]
```diff
@@ -16,8 +16,12 @@ odoo.define('web_export_view', function (require) {
             var self = this;
             this._super.apply(this, arguments);
             if (self.getParent().renderer.viewType == 'list') {
-                self.$el.find('.o_dropdown').last().append(QWeb.render('WebExportTreeViewXls', {widget: self}));
-                self.$el.find('.export_treeview_xls').on('click', self.on_sidebar_export_treeview_xls);
+                session.user_has_group('web_export_view.group_disallow_export_view_data_excel').then(function (has_group) {
```
Comment (pedrobaeza) [inline web_export_view/static/src/js/web_export_view.js:19]: Why calling again `has_group` if you have already called `user_has_group`?

Comment (chienandalu) [reply web_export_view/static/src/js/web_export_view.js:19]: No, `user_has_group` returns a deferred object that we need to resolve in the proper method. `has_group` is the boolean value we're evaluating after all.

Comment (pedrobaeza) [reply web_export_view/static/src/js/web_export_view.js:19]: Sorry for the noise. My lack of JS knowledge...

**Review by pedrobaeza — APPROVED** [review]
Add yourself to contributors in the README.

**Review by tarteo — APPROVED** [review]

**Pull request merged**